### PR TITLE
copy(fxa-auth-server): update verifyLoginCode copy

### PR DESCRIPTION
**Pull request by millsoper**
## Because:

* We have final copy for verifyLoginCode email

## This commit:

* Updates copy and tests for email

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Variables:
 #  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
-verifyLoginCode-subject-line = Sign-in code for { $serviceName }
-verifyLoginCode-title = Is this you signing in?
-verifyLoginCode-prompt-2 = If yes, here is the confirmation code:
+verifyLoginCode-subject-line-2 = Approve sign-in to { $serviceName }
+# Variables:
+#  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
+verifyLoginCode-title-2 = Did you sign in to { $serviceName }?
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl:6]: nit: "sign-in" isn't hyphenated here, but it is above on L3 (which seems to be an existing usage).
Not sure which one matches our style guide, but hyphenated seems more common based on my quick 👀 through the diff.

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.mjml [hunk 2]
```diff
@@ -5,7 +5,10 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-header">
-      <span data-l10n-id="verifyLoginCode-title">Is this you signing in?</span>
+      <span data-l10n-id="verifyLoginCode-title-2" data-l10n-args="<%= JSON.stringify({ serviceName }) %>">Did you sign in to <%- locals.serviceName %>?</span>
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.mjml:8]: nit: "sign-in" not hyphenated here either (but is below on L11 in this file).

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.txt [hunk 3]
```diff
@@ -1,11 +1,13 @@
-verifyLoginCode-title = "Is this you signing in?"
+verifyLoginCode-title-2 = "Did you sign in to <%- serviceName %>?"
```
Comment (pdehaan) [inline packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.txt:1]: hyphenate "sign-in" here too?

File: packages/fxa-auth-server/test/local/senders/emails.ts [hunk 4]
```diff
@@ -873,16 +873,16 @@ const TESTS: [string, any, Record<string, any>?][] = [
...
       ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('verifyLoginCode') }],
       ['X-Signin-Verify-Code', { test: 'equal', expected: MESSAGE.code }],
       ['X-Template-Name', { test: 'equal', expected: 'verifyLoginCode' }],
       ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.verifyLoginCode }],
     ])],
     ['html', [
-      { test: 'include', expected: 'Sign-in code for Mock Relier' },
-      { test: 'include', expected: 'Is this you signing in?' },
+      { test: 'include', expected: 'Approve sign-in to Mock Relier' },
+      { test: 'include', expected: 'Did you sign in to Mock Relier?' },
```
Comment (pdehaan) [inline packages/fxa-auth-server/test/local/senders/emails.ts:885]: needs to be hyphenated here and below too, but the tests will fail once/if we modify reference string anyways.

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl (hunk 1 above)
Comment (flodolo) [reply packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl:6]: "sign in" = verb
"sign-in" = noun

This is correct.

Comment (bcolsson) [reply packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl:6]: The usage in this file looks good to me. When used as a noun, sign in is typically hyphenated (approve the sign-in to your account) whereas when used as a verb (enter password to sign in to your account) it should not be hyphenated.

File: packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.mjml (hunk 2 above)
Comment (millsoper) [reply packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.mjml:8]: same as above -- I think this is a verb vs noun use

**Pull request merged**